python/lsst/ap/association/packageAlerts.py

```diff
@@ -372,3 +466,62 @@ def streamCcdDataToBytes(self, cutout):
             cutout.write(streamer, format="fits")
             cutoutBytes = streamer.getvalue()
         return cutoutBytes
+
+    def _serializeAlert(self, alert, schema=None, schema_id=0):
+        """Serialize an alert to a byte sequence for sending to Kafka.
+
+        Parameters
+        ----------`
```

Comment:
Excess ` at the end

Reply:
If people are interested, I have been experimenting with numpydoc validation in middleware packages and can point people at that. Also, the velin package can fix some problems automatically.

Reply:
@timj Yes I definitely would be interested in that!

Reply:
`velin` fixes automatically both of the problems that @parejkoj points out above. It also fixes some other indent issues and variable name and section headings. We don't install `velin` in the shared stack by default (it's not in rubin-develop-env but maybe it could be cc/ @ktlim ).

`numpydoc` is also not in the rubin env but has many more extensive checks. It also needs to have a lot of configuration. The quickest way to see what you need to do is to look at lsst/daf_butler#922 and see the changes I made to `pyproject.toml` and the github workflows.

You can run the validation with:

```bash
$ python -m numpydoc.hooks.validate_docstrings $(find python -name "*.py")
```
after you've installed `numpydoc`. This will definitely need some tuning in your config files to keep the checks you think are useful.